asv_bench/benchmarks/io/excel.py

```diff
@@ -40,7 +40,7 @@ def time_write_excel(self, engine):
 
 class ReadExcel:
 
-    params = ["xlrd", "openpyxl", "odf"]
+    params = ["openpyxl", "odf"]
```

Comment:
We could also rather keep this, but with writing an xls file (in addition to the xlsx file) to read with xlrd?